include/xeus-zmq/xserver_zmq.hpp

```diff
@@ -39,6 +40,8 @@ namespace xeus
 
         ~xserver_zmq() override;
 
+        zmq::multipart_t create_iopub_welcome_wire_msg(const std::string& topic);
```

Comment:
This could be renamed into `serialize_iopub`, accepting a `xpub_message` and returning a `zmq_multipart_t`; this mehtod would simply forward the call to the serializer.